fabfile/config.py

```diff
@@ -12,6 +12,11 @@
 HOME_DIRECTORY = HOME_DIR
 RESULTS_DIRECTORY = os.path.join(HOME_DIR, 'results')
 CITUS_INSTALLATION = os.path.join(HOME_DIR, 'citus-installation')
+RELATIVE_REGRESS_PATH = 'src/test/regress'
+# valgrind test variables
+VALGRIND_LOGS_FILE = 'logs.txt'
```

Comment:
`logs.txt` doesnt reveal what this file contains. Maybe rename it to `valgrind_logs.txt`

Reply:
that makes sense